waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx

```diff
@@ -22,17 +22,27 @@ const FooterBackground = styled.div`
   padding: 2.5rem 6rem 7.5rem;
   transition: all 0.3s;
   border: none !important;
+  @media screen and (max-width: 1024px) {
+    min-height: 15vh;
+  }
   @media screen and (max-width: 425px) {
     padding: 1rem;
-    min-height: 70vh;
+    min-height: 30vh;
     overflow: hidden;
   }
+  @media screen and (max-width: 375px) {
+    min-height: 35vh;
+  }
 `;
 
 const LogoAndContactWrapper = styled.div`
   display: flex;
   flex-direction: row;
   justify-content: start;
```

Comment:
Can you add `align-items: center` after this line to make sure that the logo and "Contact Us" section are aligned vertically? This can be kept outside of `@media` since they should be aligned in desktop view as well.

Reply:
By putting `align-items: center` outside @media, it goes off the design for the desktop view I think. For now, I have it set to be inside the @media 
<img width="1386" alt="Screen Shot 2022-03-16 at 12 54 09 AM" src="https://user-images.githubusercontent.com/84100023/158519407-6ff3bc15-8a10-4623-bce8-3be708a65820.png">
.

Reply:
What I was hoping to do by setting `align-items: center` here was to have the logo and contact section aligned vertically, because when I saw it on my laptop they were slightly offset in both desktop and mobile views. I didn't want the horizontal position of the logo+contact on the page changed at all. But looking at your recent changes, it looks like you manually aligned them anyways (seems like the SVG itself has some weird whitespace, so that should be fine). So how it is right now should be fine.